README.md

```diff
@@ -650,7 +655,7 @@ $ oc expose svc el-vote-app
 
 ## Configuring GitHub WebHooks
 
-Now we need to configure webhook-url on [backend](https://github.com/openshift-pipelines/vote-api) and [frontend](https://github.com/openshift-pipelines/vote-ui) source code repositories with the Route we exposed in the previously.
+Now we need to configure webhook-url on [backend](https://github.com/openshift-pipelines/vote-api/tree/release-tech-preview-1) and [frontend](https://github.com/openshift-pipelines/vote-ui/tree/release-tech-preview-1) source code repositories with the Route we exposed in the previously.
```

Comment:
Please do change (we exposed in the previously -> we exposed previously)

Reply:
is it not better we do this in master?

Reply:
both ? :stuck_out_tongue_closed_eyes: